graph/push.go

```diff
@@ -517,16 +514,16 @@ func (s *TagStore) CmdPush(job *engine.Job) error {
 		return err
 	}
 
-	r, err := registry.NewSession(authConfig, registry.HTTPRequestFactory(metaHeaders), endpoint, false)
+	r, err := registry.NewSession(imagePushConfig.AuthConfig, registry.HTTPRequestFactory(imagePushConfig.MetaHeaders), endpoint, false)
 	if err != nil {
 		return err
 	}
 
-	reposLen := 1
-	if tag == "" {
-		reposLen = len(s.Repositories[repoInfo.LocalName])
-	}
-	job.Stdout.Write(sf.FormatStatus("", "The push refers to a repository [%s] (len: %d)", repoInfo.CanonicalName, reposLen))
+	//reposLen := 1
+	//if imagePushConfig.Tag == "" {
+	//	reposLen = len(s.Repositories[repoInfo.LocalName])
+	//}
+	//imagePushConfig.OutStream.Write(sf.FormatStatus("", "The push refers to a repository [%s] (len: %d)", repoInfo.CanonicalName, reposLen))
```

Comment:
I could use some advice here.
The reason why i did this is to pass the test-integration-cli `TestPushUntagged` & `TestPushBadTag`. 
Given the prompts didnt match the expected results of those two previously mentioned test cases and MAY not be necessary, i simply drop them :(